xrootd/authorization.go

```diff
@@ -134,23 +134,30 @@ func EmitAuthfile(nsAds []director.NamespaceAd) error {
 	sc := bufio.NewScanner(strings.NewReader(string(contents)))
 	output := new(bytes.Buffer)
 	foundPublicLine := false
-	for sc.Scan() {
-		lineContents := sc.Text()
-		words := strings.Fields(lineContents)
-		if len(words) >= 2 && words[0] == "u" && words[1] == "*" {
-			output.Write([]byte("u * /.well-known lr " + strings.Join(words[2:], " ") + "\n"))
-			foundPublicLine = true
-		} else {
-			output.Write([]byte(lineContents + "\n"))
+	if nsAds == nil {
```

Comment:
This could probably use some explaining.  It looks like the input auth file is ignored `nsAds` isn't null.  Why?

To support some legacy use cases, we'll need to merge in an authfile for the cache.